src/tools/rbd/OptionPrinter.cc

```diff
@@ -74,9 +74,23 @@ void OptionPrinter::print_detailed(std::ostream &os) {
     std::cout << std::endl;
   }
 
+  std::vector<std::string> deprecated_set = {"order", "new-format"};
```

Comment:
I would prefer that you don't add a list of deprecated options. Instead, if the help text includes `deprecated`, don't print the line. Similarly, upon parsing the command line, if someone specifies a command w/ "deprecated" in the help test, print the warning.